package.json

```diff
@@ -24,7 +24,7 @@
   "devDependencies": {
     "grunt": "^1.0.1",
     "grunt-contrib-cssmin": "^2.2.0",
-    "grunt-contrib-uglify": "^3.0.1",
+    "grunt-contrib-uglify": "gruntjs/grunt-contrib-uglify#harmony",
```

Comment:
Yeah that is a problem: At the moment only the [grunt-contrib-uglify#harmony](https://github.com/gruntjs/grunt-contrib-uglify/tree/harmony) branch is compatible with JavaScript ES2015 syntax. I checked today, if they merged the branch yet, but unfortunately they did not.